Makefile

```diff
@@ -40,8 +40,9 @@ vendor-tidy:
 vendor:
 	go mod vendor
 
+$(GOPATH)/bin/golint:
+	GO111MODULE=off go get -u golang.org/x/lint/golint
 lint:
```

Comment:
Don't we need to include the `$(GOPATH)/bin/golint` task in this so that golint is downloaded when someone call `make lint`?

Reply:
sorry seems it's missing. just because on my dev machine it would download a new lint version that will stuck, so I made this change according to operator.